### ISSUE TITLE
[Feature Request/Enhancement] Autohide Tabbar like Firefox

### ISSUE THREAD

**Issue by GunGunGun** [notepad-plus-plus/notepad-plus-plus#12801]
### Description of the Issue
I use Multi-line Tabbar, and I usually open a lot of document for many project, it would be greate to be able to autohide the tabbar to save space, will be a godsend to me because Multi-line Tabbar uses a lot of space and Single-line Tabbar is unusable in my case, thanks!

I want it to be able to autohide like this, like Firefox can: https://gfycat.com/MasculineImaginativeHorseshoebat
https://gfycat.com/ShockingPreciousCoqui

Best regards,
Gun

**Comment by alankilborn** [comment]
I don't know how Firefox works, but I presume when you move your mouse into the space where a collapsed tab bar is, it expands (temporarily) to show you all of the tabs and you can then make a choice of tab, to have the tab bar collapse again with the new tab active.

Sounds kind of cool, perhaps a good addition to N++.

**Comment by GunGunGun** [comment]
> I don't know how Firefox works, but I presume when you move your mouse into the space where a collapsed tab bar is, it expands (temporarily) to show you all of the tabs and you can then make a choice of tab, to have the tab bar collapse again with the new tab active.
> 
> Sounds kind of cool, perhaps a good addition to N++.

Yeah, something like that, because Notepad++ supports many programming language,  that's why I believe most of us open a lot of projects to work with (as for me I'm a Python/Javascript/Autohotkey/LUA developer, I need to open all my project in Notepad++ for work), and because of that our tab stack keeps expanding, 10 then 30, 50, then 70... And multi-line tabbar will take 50% of screen width and single-line tabbar is just unusable because scrolling to change tabs will take a lot of time. So I think being able to autohide tabbar will be a big game changer.

**Comment by alankilborn** [comment]
Currently, you could hide the tab bar and use the _NavigateTo_ plugin for switching between tabs.

**Comment by GunGunGun** [comment]
> _NavigateTo_

Thanks a lot, I tried it and it's almost perfect for my needs, would be greater if I can also autohide but I can live with a mapped Alt + Z shortcut to show/hide it and find another way to achieve this or wait for new features coming later on (if it's possible).

Still, really thank you!
